pallets/settlement/src/lib.rs

```diff
@@ -602,6 +613,7 @@ decl_module! {
         /// * `trade_date` - Optional date from which people can interact with this instruction.
         /// * `value_date` - Optional date after which the instruction should be settled (not enforced)
         /// * `legs` - Legs included in this instruction.
+        /// * `memo` - Optional memo field for this instruction.
```

Comment:
This extrinsic doesn't have the `memo` parameter anymore.  So this needs to be removed.